twitter-langchain/twitter_langchain/twitter_api_wrapper.py

```diff
@@ -24,13 +23,6 @@ def validate_environment(cls, values: dict) -> Any:
         access_token = get_from_dict_or_env(values, "twitter_access_token", "TWITTER_ACCESS_TOKEN")
         access_token_secret = get_from_dict_or_env(values, "twitter_access_token_secret", "TWITTER_ACCESS_TOKEN_SECRET")
 
-        try:
-            import tweepy
-        except Exception:
-            raise ImportError(
-                "Tweepy Twitter SDK is not installed. " "Please install it with `pip install tweepy`"
-            ) from None
-
         client = tweepy.Client(
```

Comment:
this validation should stay I think. The one I wanted to remove was the extra import in `run_action`

Reply:
resolved in most recent commit